projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_action-strip.scss

```diff
@@ -13,12 +12,10 @@
 /// @prop {Map} background [color: ('grays', 100] - The color used for the action strip component content background.
 /// @prop {Map} actions-background [color: ('grays', 200)] - The color used for actions background.
 /// @prop {Map} delete-action [color: ('error')] - The color used for the delete icon in action strip component.
-/// @prop {Number} border-radius [1] - The border radius used for action strip actions container. Can be a fraction between 0 and 1, pixels, or percent.
+/// @prop {Number} actions-border-radius [1] - The border radius used for action strip actions container. Can be a fraction between 0 and 1, pixels, or percent.
```

Comment:
Hm, what necessitates the change? If this must be changed here and in the theme then a migration should be written.